dashboard/config/scripts_json/k5-onlinepd-2021.script_json

```diff
@@ -4,11 +4,12 @@
     "wrapup_video_id": null,
     "login_required": true,
     "properties": {
+      "is_course": true,
       "is_migrated": true,
       "version_year": "2021"
     },
     "new_name": null,
-    "family_name": null,
+    "family_name": "k5-onlinepd",
```

Comment:
I think we had said we did not want family names to include years. That is more consistent with the new pattern we are using when you create a new script. Also there are no other scripts that use "k5-onlinepd" or "peru" as their family name right now so it will not group anything together.

Reply:
There are other k5-onlinepd scripts but they don't have a family name or version year set. I don't know if we want to set these fields on all or none of them, but we should probably be consistent.

Conversely, there's only one "peru" script, so I think it's fine for that family name to be set.

Reply:
>> I think we had said we did not want family names to include years. That is more consistent with the new pattern we are using when you create a new script. 

Good point, some of this is coming back to me now and I think we did decide this for new scripts. 

> > Also there are no other scripts that use "k5-onlinepd" or "peru" as their family name right now so it will not group anything together.

> There are other k5-onlinepd scripts but they don't have a family name or version year set. I don't know if we want to set these fields on all or none of them, but we should probably be consistent.

> Conversely, there's only one "peru" script, so I think it's fine for that family name to be set.

Bethany's comment makes sense to me. ideally our old scripts would follow our new pattern and we wouldn't have k5-onlinepd-xxxx unless they were sharing a version dropdown. but since we do have them and they are not currently sharing a dropdown, the options seem to be:
1.  set the family name to the script name and use a version year of "unversioned" (easiest?)
2. clear the family name and version year (I can't remember if this is still a legit option?)
3. get buy-in to have these courses share the version dropdown, and set the family name to k5-onlinepd for all

I have no idea if (3) would make a difference to the PL team or not, but if it is not too bad for them, then maybe we can do it to get all our scripts onto the new system.

Reply:
Because none of the PL courses are launched the assignment dropdown isn't showing for any of them currently and won't be unless we launch (mark as preview or stable). I'm on board to set the family name for all the k5-onlinepd courses to the same thing and since they are not launched don't see any downside. Is there anything besides the assignment/redirect that should be considered?

Reply:
Also this is blocking marketing from making changes so I would like to get it in today. We could hold the change on this family name if that feels like it needs more conversation

Reply:
Yeah my vote is to include all the changes here except k5-onlinepd and do that as a separate PR, so we can continue the conversation there. I think we're in agreement that the rest of the changes look good

Reply:
> Yeah my vote is to include all the changes here except k5-onlinepd and do that as a separate PR, so we can continue the conversation there. I think we're in agreement that the rest of the changes look good

this generally sounds right to me, to unblock marketing today and take a bit longer to settle on a path for k5-onlinepd. just confirming though that reverting changes to k5-onlinepd-2021.script_json would leave it uneditable until the second PR goes in? If we want to avoid that, we could just set the family names to the script name for now, or clear the version years. I don't think either of those options would have any un-undoable consequences (although I'm still not sure I remember whether clearing both family name and version year is a viable option).

Reply:
It should be fine to edit it as long as I remove the is_course setting too.

Reply:
Alright - I have removed the change for k5-onlinepd and moved it here: https://github.com/code-dot-org/code-dot-org/pull/44232.

Reply:
The same things is also true for vpl scripts so going to move those out to the future PR too

Reply:
> The same things is also true for vpl scripts so going to move those out to the future PR too

sorry I missed this. sounds good 👍